docs/what-is-this.adoc

```diff
@@ -28,7 +28,7 @@ and `maven-failsafe-plugin`.
 
 === Minimum requirements
 
-Java:: Requires Java 8.
+Java:: Requires Java 8. In case of using <<Affected>> strategy, Java 9 is not supported yet.
 
```

Comment:
This looks little confusing. Are we supporting java9 for other strategies. I don't remember anyone tested smart-testing tool with java9